src/Sculpin/Bundle/SculpinBundle/Command/InitCommand.php

```diff
@@ -0,0 +1,218 @@
+        $this->createSiteConfigFile($projectDir, $title, $subTitle, $output);
+
+        // 4. Create source folder (with or without posts) and the very first basic entry in the source folder
+        $this->createSourceFolder($projectDir, $output);
+
+        $output->writeln('<info>Success!</info>');
+        $output->writeln('Run "sculpin generate --watch --server" to see your static site in action.');
+
+        return self::COMMAND_SUCCESS;
+    }
+
+    protected function ensureCleanSlate(string $projectDir, OutputInterface $output): bool
+    {
+        $fs = new Filesystem();
+        if ($fs->exists($projectDir . '/app')) {
+            $output->writeln('<info>/app folder exists.</info>');
+
+            return false;
+        }
+
+        if ($fs->exists($projectDir . '/source')) {
+            $output->writeln('<info>/source folder exists.</info>');
+
+            return false;
+        }
+
+        return true;
+    }
+
+    protected function createDefaultKernel(string $projectDir, OutputInterface $output): bool
+    {
+        $contents = <<<EOF
+<?php
+
+class SculpinKernel extends \Sculpin\Bundle\SculpinBundle\HttpKernel\AbstractKernel
+{
+    protected function getAdditionalSculpinBundles(): array
+    {
+        return [
+//            'App\\Bundle\\ExampleBundle\\AppExampleBundle'
+        ];
+    }
+}
+
+EOF;
+        $this->createFile($projectDir . '/app/SculpinKernel.php', $contents);
+
+        return true;
+    }
+
+    protected function createSiteKernelFile(string $projectDir, OutputInterface $output): bool
+    {
+        $contents = <<<EOF
+sculpin_content_types:
+    posts:
+      enabled: false
+
+EOF;
+        $this->createFile($projectDir . '/app/config/sculpin_kernel.yml', $contents);
+
+        return true;
+    }
+
+    protected function createSiteConfigFile(
+        string $projectDir,
+        string $title,
+        string $subTitle,
+        OutputInterface $output
+    ): bool {
+        $contents = <<<EOF
+title: "$title"
+subtitle: "$subTitle"
+google_analytics_tracking_id: ''
+url: ''
+
+EOF;
+        $this->createFile($projectDir . '/app/config/sculpin_site.yml', $contents);
+
+        return true;
+    }
+
+    protected function createSourceFolder(string $projectDir, OutputInterface $output): bool
+    {
+        $fs = new Filesystem();
+
+        $fs->dumpFile(
+            $projectDir . '/source/index.md',
+            <<<EOF
+---
+layout: default
+---
+
+<h1>Welcome to {{site.title}}</h1>
+
+EOF
+        );
+
+        $fs->dumpFile(
+            $projectDir . '/source/_views/default.html',
+            <<<EOF
+<html>
+<head><title>{{site.title}}</title></head>
+<body>
+{% block content_wrapper %}{% block content '' %}{% endblock content_wrapper %}
+</body>
+</html>
+
+EOF
+        );
+
+        return true;
+    }
+
+    protected function createFile(string $path, string $contents): void
+    {
+        $fs = new Filesystem();
+        $fs->dumpFile($path, $contents);
+    }
+}
```

Comment:
Have you considered `mirror` method on `FileSystem`?

https://github.com/Symplify/Symplify/pull/1285/files#diff-46fa2040cb0556b451df6dc4391c23beR42